src/model.js

```diff
@@ -0,0 +1,58 @@
+export default class Model {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  setStorage = (todos) => {
+    // this.tasks.forEach(element => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+    // });
```

Comment:
- [ ] [Optional] I would advise you remove commented codes in order to make your code more clean

Reply:
I will make sure I do that tomorrow. Thank you